content/trees/navigation/en/session_replay.yaml

```diff
@@ -13,17 +13,47 @@ tree:
     entry: 467a0fe0-6ad9-4375-96a2-eea5b04a7bcf
     children:
       -
-        id: 6573c21d-8fc2-4206-96bd-5041b7313ad4
-        entry: 99b87b72-07f8-4e08-9dcd-348f3c8976f3
+        id: 26be92c5-cd46-4ff7-9a45-9194697c532a
+        title: Web
+        children:
+          -
+            id: 2a9877d6-5e50-43d9-a1e0-15862a4279b5
+            entry: f4c61e43-7a55-43a3-9dbd-f1ecb2502774
+          -
+            id: 6573c21d-8fc2-4206-96bd-5041b7313ad4
+            entry: 99b87b72-07f8-4e08-9dcd-348f3c8976f3
+          -
+            id: 3706eba9-c43a-4725-a801-bf9eac846f49
+            entry: d225a8c4-c504-4cbe-b4f3-8709ab43fdc1
+          -
+            id: 25c81e16-709f-42e1-9655-bbb6eb0fc606
+            entry: 5416910c-32f8-4e80-be0f-88a10509c029
       -
-        id: 2a9877d6-5e50-43d9-a1e0-15862a4279b5
-        entry: f4c61e43-7a55-43a3-9dbd-f1ecb2502774
-      -
-        id: 3706eba9-c43a-4725-a801-bf9eac846f49
-        entry: d225a8c4-c504-4cbe-b4f3-8709ab43fdc1
+        id: 3890b43d-83e6-40d5-b938-b82ea1ee9fcc
+        title: iOS
```

Comment:
I don't think we want to add iOS and Android to the tree yet. This linking will happen when we do a public launch in Jan.